packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -162,6 +162,12 @@ export type CreateSavedChartOrVersionEvent = BaseTrack & {
             yAxisCount: number;
             seriesCount: number;
             seriesTypes: CartesianSeriesType[];
+            referenceLinesCount?: number;
+            margins?: string;
+            showLegend?: boolean;
+        };
+        table?: {
+            conditionalFormattingRulesCount?: number;
```

Comment:
We should make these required so the count values default to 0 instead of undefined.